### PR TITLE
docs: Update validator-faq.md for grammar, clarity, tense, and ATOM

### PR DESCRIPTION
**Pull request by barriebyron**
more editorial love for this essential content

Closes my own yearning for accurate and current content 

## Description

ATOM (not Atoms) is the accurate token name

- use present tense, per https://developers.google.com/style/tense
- avoid unclear antecedents ("it" and "they" without a subject), I know this writing clarity is a real challenge and we all are guilty
- avoid should (too ambiguous)
- avoid the collective first person "we" and "let's" 
- use descriptive link text https://developers.google.com/style/link-text 
- and a little more editorial love for our future amazing validators and delegators who want to learn about validators

As I mentioned in https://github.com/cosmos/gaia/pull/1086, I was asked to share information about how to select a validator and came across the opportunities along with https://github.com/cosmos/gaia/pull/1086 

This topic will benefit from a thorough review for accuracy and currency. Who is the right subject matter expert for this review?

Seems like information overlaps with #1086 I recommend not duplicating content

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/main/README.md#merging-a-pr))

### CODE REVIEW

**Comment by barriebyron** [comment]
hi gaia friends! Who can review and approve this PR? @jgimeno @okwme @zmanian

File: docs/validators/validator-faq.md [hunk 1]
```diff
@@ -10,31 +10,31 @@ This is work in progress. Mechanisms and values are susceptible to change.
...
 
-### What is 'staking'?
+### What is staking?
 
-The Cosmos Hub is a public Proof-Of-Stake (PoS) blockchain, meaning that the weight of validators is determined by the amount of staking tokens (Atoms) bonded as collateral. These Atoms can be self-delegated directly by the validator or delegated to them by other Atom holders.
+The Cosmos Hub is a public Proof-Of-Stake (PoS) blockchain, meaning that the weight of validators is determined by the amount of staking tokens (ATOM) bonded as collateral. These ATOM can be self-delegated directly by the validator or delegated to the validator by other ATOM holders.
```
Comment (yaruwangway) [inline docs/validators/validator-faq.md:19]: These ATOM &#8594; These ATOMs

File: docs/validators/validator-faq.md [hunk 2]
```diff
@@ -10,31 +10,31 @@ This is work in progress. Mechanisms and values are susceptible to change.
...
+Because delegators share revenue with their validators, they also share risks. If a validator misbehaves, each of their delegators are partially slashed in proportion to their delegated stake. This penalty is one of the reasons why delegators must perform due diligence on validators before delegating. Spreading their stake over multiple validators is another layer of protection.
 
-Delegators play a critical role in the system, as they are responsible for choosing validators. Being a delegator is not a passive role: Delegators should actively monitor the actions of their validators and participate in governance. For more, read the [delegator's faq](https://hub.cosmos.network/main/delegators/delegator-faq.html).
+Delegators play a critical role in the system, as they are responsible for choosing validators. Being a delegator is not a passive role. Delegators must actively monitor the actions of their validators and participate in governance. For details on being a delegator, read the [Delegator FAQ](https://hub.cosmos.network/main/delegators/delegator-faq.html).
```
Comment (yaruwangway) [inline docs/validators/validator-faq.md:37]: Being a delegator is not a passive role. &#8594; Delegator is not a passive role

File: docs/validators/validator-faq.md (hunk 1 above)
Comment (barriebyron) [reply docs/validators/validator-faq.md:19]: Hi @yaruwangway that's what I used to think too! 
see the approved [Cosmos Word List](https://docs.google.com/spreadsheets/d/15raX8lfzzeeifGrDiXSFTHaEDbsMVlFJzKgnNelihjs/edit#gid=0) that we work from for our docs under the tendermint.com umbrella, including the

Comment (yaruwangway) [reply docs/validators/validator-faq.md:19]: @barriebyron, thanks for sharing!

Comment (okwme) [reply docs/validators/validator-faq.md:19]: wow this list is amazing @barriebyron !!!

File: docs/validators/validator-faq.md (hunk 2 above)
Comment (okwme) [reply docs/validators/validator-faq.md:37]: i think both make sense, but barrie's sounds more natural to my ear. let's stay with her suggestion

**Review by okwme — APPROVED** [review]
🙏

**Pull request merged**